default/stringtables/en.txt

```diff
@@ -339,11 +339,14 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. The amount to transfer and how many times the transfer should repeat can be specified. PP allocated to the transfer are added to the imperial stockpile.
 
-Only transfers PP that are available at this planet; PP allocation on planets that are not supply-connected to here will not be affected by this project.
+Only PP that are produced in the same supply-group are allocated to the stockpile transfer project. PP already in the imperial stockpile are not allocated to the stockpile transfer project.
```

Comment:
ending two sentences right after eachother with "allocated to the stockpile transfer project" is awkward. Maybe:

Only PP that are generated in the same supply group (and none from the imperial stockpile) can be allocated to an Stockpile Transfer project.